install

```diff
@@ -697,10 +698,12 @@ verbose "Using distribution id \"$dist\", release code \"$code\""
 systemInitDetect
 
 if $is_centos_dist; then
+    python_bin="python3"
     apache_bin="httpd"
     apache_service="httpd"
     web_user="${web_user:-apache}"
 else
+    python_bin="python"
```

Comment:
This should probably be python3 everywhere. I think I manually replaced things with python3 in the script elsewhere

Reply:
Seeing as we don't support python 2

Reply:
Look back at this I have no clue how I introduced it... I'll do some testing based on the Debian vagrant machines before I fix it.

Reply:
I guess [PEP 394](https://www.python.org/dev/peps/pep-0394/) got me confused. CentOS now uses 3.8 since I'll need that to install psycopg2 for the api later on. Debian is untouched and uses `python3` again.

Reply:
Debian systems are unlikely to ever use `python` for `python3`. It may be offered through alternatives, but the Python team's plan (as I recall) is to have `python2` and `python3` and not install a `python` binary at all.